packages/v1-sdk/src/entities/pair.ts

```diff
@@ -158,76 +184,121 @@ export class Pair {
 
   public getOutputAmount(inputAmount: CurrencyAmount<Token>): [CurrencyAmount<Token>, Pair] {
     invariant(this.involvesToken(inputAmount.currency), 'TOKEN');
-    if (this.reserve0.quotient === ZERO || this.reserve1.quotient === ZERO) {
-      throw new InsufficientReservesError();
-    }
 
-    const inputReserve = this.reserveOf(inputAmount.currency);
-    const outputReserve = this.reserveOf(inputAmount.currency.equals(this.token0) ? this.token1 : this.token0);
+    if (this.isStabilityFund) {
+      // this.token1 is always bnUSD
+      if (inputAmount.currency.symbol === 'bnUSD') {
+        // bnUSD -> USDC
+        // apply fee 0.1%
+        const outputAmount = CurrencyAmount.fromCurrencyAmount(inputAmount, this.token0).multiply(
+          new Fraction(999, 1000),
+        );
+
+        if (outputAmount.quotient > this.reserve0.quotient) {
+          throw new InsufficientInputAmountError();
+        }
 
-    if (this.isQueue) {
-      if (inputAmount.currency.address === NULL_CONTRACT_ADDRESS) {
-        // ICX -> sICX
-        const numerator = inputAmount.numerator * outputReserve.quotient;
-        const denominator = inputAmount.denominator * inputReserve.quotient;
-        const outputAmount = CurrencyAmount.fromRawAmount(outputReserve.currency, numerator / denominator);
-        return [outputAmount, new Pair(inputReserve.add(inputAmount), outputReserve.subtract(outputAmount), {})];
+        return [outputAmount, this];
       } else {
-        const numerator = inputAmount.numerator * _99 * outputReserve.quotient;
-        const denominator = inputAmount.denominator * _100 * inputReserve.quotient;
-        const outputAmount = CurrencyAmount.fromRawAmount(outputReserve.currency, numerator / denominator);
-        return [outputAmount, new Pair(inputReserve.add(inputAmount), outputReserve.subtract(outputAmount), {})];
+        // USDC -> bnUSD
+        return [CurrencyAmount.fromCurrencyAmount(inputAmount, this.token1), this];
+      }
+    } else {
```

Comment:
could you remove else statement? the main purpose is to reduce the git difference.

Reply:
done